### ISSUE TITLE
TS Path alias option?

### ISSUE THREAD

**Issue by brian428** [mgechev/angular-seed#2226]
Minko, would you have any interest in a PR that lets you specify a TS path alias for the app? This would allow you to add a TS path mapping to tsconfig and use "absolute" imports instead of relative imports. E.g.:

Instead of: 
`import { NameListService } from '../shared/name-list/name-list.service';`

You could do something like: 
`import { NameListService } from '@app-name/shared/name-list/name-list.service';`

Once my app got big, all the "../../../" in the imports got on my nerves heh. But maybe I'm the only one who finds the relative imports to be ugly. 

Basically, I'd probably try to add a new property to SeedConfig where an alias could be specified from ProjectConfig that matches whatever path alias you added to `tsconfig.json`, such as:

`TS_PATH_ALIAS = "@app-name"`

If a value is set, the build would conditionally add a matching path to the SystemJS config. 

Let me know what you think.

**Comment by mgechev** [comment]
I'd say it's a good idea to have this in a blog post, wiki, or the readme. Let's keep it out of the seed because it may make the project feel like even a bigger magic.
